Add compatibilities and bump patch version

torfjelde opened the pull request:
Depends on the following PRs:
- [x] https://github.com/FluxML/Tracker.jl/pull/55
- [x] https://github.com/FluxML/Zygote.jl/pull/398

`DistributionsAD` currently breaks for newer versions of `Zygote` and other related/dependent packages, including `Turing`. The lower-bounds in this PR provide a fix.

UPDATE: both PRs have been merged, now we're just waiting for relases to happen.

torfjelde commented:
It seems lie builds are failing because the Travis build isn't using the most up-to-date package registry. Locally, doing `] up` results in `Zygote@0.4.1` while in the Travis log you can see `Zygote@0.3.4` is being used.

mohamed82008 commented:
Please update the Manifest file.

torfjelde commented:
> Please update the Manifest file.

Hah, just edited my comment saying the same thing. You want me to update or remove the Manifest.toml?

mohamed82008 commented:
Hmm, I would rather keep it to be able to easily reproduce Travis failures when they happen, but it's a matter of taste.

torfjelde commented:
Seems like good reasoning to me :+1: Updated and tests are running now. Should pass.

Worth noting that the version-bounds I've added, especially for Zygote. The reason for this is the breaking changes introduced in `Zygote@0.4`.

mohamed82008 commented:
Cool thanks for this PR!

torfjelde commented:
This should be ready now:)

mohamed82008 commented:
Thanks @torfjelde !
